cursorless-talon/src/cheatsheet/sections/actions.py

```diff
@@ -16,7 +16,7 @@ def get_actions():
         "rewrap",
     ]
     simple_actions = {
-        key: value
```

Comment:
This change is the one that is most controversial. It's ok with me, but there were objections to how much noise it adds to the cheatsheet cc/ @AndreasArvidsson

Reply:
Talked with Tara Roys and a few other users at the cursorless meetup on 4/10 and they all were in favor of adding the T's for clarity.

Reply:
If people prefer it that way: go ahead :)